app/src/ui/welcome/configure-git.tsx

```diff
@@ -12,9 +13,23 @@ interface IConfigureGitProps {
 
 /** The Welcome flow step to configure git. */
 export class ConfigureGit extends React.Component<IConfigureGitProps, {}> {
+  private configureGitElement = React.createRef<HTMLElement>()
+
+  public componentDidMount() {
+    if (this.configureGitElement.current == null) {
+      return
+    }
+
+    focusFirstSuitableChild(this.configureGitElement.current)
```

Comment:
I reused this logic from our dialogs here with a tweak to focus on the radio buttons (I think that makes sense? but then I don't understand why we exclude radio buttons in the dialogs). I think it should be used on the other screens as well because the are multiple  a11y linters that say using the `autofocus` property is a violation. But.. have yet to look into the why behind that violation since we need to focus the first focusable thing anyhow.

Reply:
So.. investigated the `autofocus` issue and it was just false positives.. so this approach was probably overkill, granted it does have the benefit of if this form changes the first one will always be selected.

Reply:
Reverted